### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by Forni82** [lukasdotcom/fantasy-manager#710] [Q&A]
Hello, how are you?
First of all, great job with your fantasy league. 
I'm looking for something similar to this, but my notions with all these topics are very basic. 
I wanted to know if it is possible to enter this league as an administrator and create one again. I have an Instagram account for the third category of Spanish soccer, and I would love to be able to create a fantastic league for them. 
It would be of great help to me if you could tell me how I should manage it. 
Many thanks for everything,

**Comment by lukasdotcom** [comment]
If I understand you correctly you want to have a league that uses custom data. This would require some programming knowledge, and for you to be able to host this yourself(probably would cost you money). You would also need to create a plugin that takes the data from wherever your getting it, and converts it to a format that this program can easily understand. There's a guide on how to do this at https://github.com/Lukasdotcom/fantasy-manager-plugin-template If you are interested in doing that and need any help on making the plugin or installation feel free to ask.

**Comment by mikihockey3-jpg** [comment]
> Hello, how are you?
> 
> First of all, great job with your fantasy league. 
> 
> I'm looking for something similar to this, but my notions with all these topics are very basic. 
> 
> I wanted to know if it is possible to enter this league as an administrator and create one again. I have an Instagram account for the third category of Spanish soccer, and I would love to be able to create a fantastic league for them. 
> 
> It would be of great help to me if you could tell me how I should manage it. 
> 
> Many thanks for everything,

Hi im trying to do the same thing for an italian league, did u manage to do it, do u have suggestions?

**Reply by lukasdotcom** [reply]
You generally need two parts a data source, which I can't help you with, and the actual plugin code itself. If you want to see some examples for the Bundesliga and the Premier League you can look in this folder https://github.com/lukasdotcom/fantasy-manager/tree/main/store to see the plugins I've made. Each folder contains a plugin

**Comment by mikihockey3-jpg** [comment]
The data source is manuay made by me, i have just problem with the program
not finding  the datas from the raw file

Il giorno mer 4 feb 2026 alle 19:42 Lukas Schaefer ***@***.***>
ha scritto:

**Reply by lukasdotcom** [reply]
If you have manual data what you can do is just write a function that returns that data.
```js
const Main: dataGetter = async (settings, past_data) => {
  const transfer_open = true; // Just choose if the transfer market is open
  const countdown = 0; // Doesn't really matter.
  const playerList: players[] = STATIC_PLAYER_LIST;
  const clubs: clubs[] = STATIC_CLUBS_LIST;
  return [transfer_open, countdown, playerList, clubs];
};

```
